src/device.rs

```diff
@@ -834,110 +834,287 @@ impl<'a> ProgrammableStageDescriptor {
     }
 }
 
+#[repr(u32)]
+pub enum BlendOperation {
```

Comment:
do we have to redefine these enums? they look exactly like the ones in `wgt`. Unless we are planning on adding the `Undefined = 0` variant to them, we could just keep using the `wgt` ones.

Reply:
is `#[repr(C)]` different from `#[repr(u32)]`?

Reply:
Looks like they match - https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=d3b8a1539dfe389d944dccfe8b7a6727

Reply:
So yes, we can avoid redefining these enums. I've replaced them with the ones in wgt.